src/components/QuestDetailsWrapper/index.tsx

```diff
@@ -616,7 +634,9 @@ export function QuestDetailsWrapper({
       },
       isSyncing: resyncMutation.isPending,
       chainTooltips: {},
-      isQuestsPage
+      isQuestsPage,
+      ...questDetailsParamProps,
+      i18n,
```

Comment:
we leave i18n at the end because we already merge the props i18n with the local definition:

```ts
const i18n: QuestDetailsTranslations = {
  ...questDetailsParamProps.i18n,
  ...
}
```